### DISCUSSION TITLE
Storing data of errors and make more such suggestion in practice

### DISCUSSION THREAD

**Discussion by subhodynamics** [monkeytypegame/monkeytype#4000] [Ideas] [answered]
One must be logged in, and practice with being logged in.
After each test as we get to practice the word missed and errors, then we can also store that data and make more such use of word often for a better practice of the user.
This will help the user to improve, and give the sense of a personalised environment to work on.
We can have separate section of such practice sessions kind of thing, just like adding punctuation and numbers to the word we are about to type.

**Comment by unrealapex** [comment, marked as answer]
You could use the `weakspot` funbox. It replicates the functionality you requested.

**Reply by Miodec** [reply]
Yeap, thats the solution right now. A more in depth system is planned.
